gp-includes/misc.php

```diff
@@ -736,3 +736,23 @@ function gp_get_sort_by_fields() {
 function gp_set_translations_import_max_memory_limit() {
 	return '256M';
 }
+
+/**
+ * Checks if a string is valid UTF-8.
+ *
+ * The wp_is_valid_utf8() was introduced in WordPress 6.9.0, so we need a fallback for older versions.
+ * seems_utf8() was deprecated in WordPress 6.9.0.
+ * See https://core.trac.wordpress.org/changeset/60630
+ *
+ * @since 4.1.0
+ *
+ * @param string $string The string to check.
+ * @return bool True if the string is valid UTF-8, false otherwise.
+ */
+function gp_is_valid_utf8( $string ) {
+	if ( function_exists( 'wp_is_valid_utf8' ) ) {
+		return wp_is_valid_utf8( $string );
+	} else {
+		return seems_utf8( $string );
+	}
```

Comment:
Following the pattern of other functions in this file like `gp_gmt_strtotime`:
```suggestion
	}
	return seems_utf8( $string );
```